tyk-headless/values.yaml

```diff
@@ -194,6 +236,11 @@ pump:
   nodeSelector: {}
   tolerations: []
   affinity: {}
+
+  # otherPumpBackend specifies if the backend apart from MongoDB, SQL or Prometheus is in use, e.g., Elasticsearch.
+  # Please set it to true while using other backends listed above. You can see all supported Pump backends here:
+  # https://tyk.io/docs/tyk-stack/tyk-pump/other-data-stores/
+  otherPumpBackend: false
```

Comment:
Should we remove the backend field from tyk-headless and instead use this field only to set pumpBackend?

Reply:
but then how can we understand the pump type? at the moment, we only set environment variables for SQL, MongoDB, and Prometheus pumps.

Reply:
Instead of backend we can use pumpBackend field.
In pro it makes sense to have backend so that based on the database dashboard is using we can create pump. But in headless we use it to create pump only.